src/main/java/io/nats/client/api/ServerInfo.java

```diff
@@ -142,7 +142,7 @@ private String getComparableVersion(String vString) {
             String[] v = vString.replaceAll("v", "").replaceAll("-", ".").split("\\Q.\\E");
             int at = vString.indexOf("-");
             return "" + (Integer.parseInt(v[0]) * 10000) + (Integer.parseInt(v[1]) * 100) + Integer.parseInt(v[2])
-                    + (at == -1 ? "" : vString.substring(at));
+                    + (at == -1 ? "~" : vString.substring(at));
```

Comment:
The tilde is the last ascii character so it this helps `v0.0.0` sort "newer" than `v.0.0.0-beta`